sdk/build.go

```diff
@@ -0,0 +1,5 @@
+package sdk
+
+type Build struct {
```

Comment:
Isn't BuildResult a more suitable struct for the sdk, as it is returned by the build service?

Reply:
Yes for sure